README.md

```diff
@@ -1,29 +1,21 @@
-# 🔌 create-eth Extensions
+# Randao Random Generator Extension for Scaffold-ETH 2
 
-This repository holds all the BG curated extensions for [create-eth](https://github.com/scaffold-eth/create-eth), so you can extend the functionality of your Scaffold-ETH project.
+This extension shows how to use on-chain randomness using RANDAO for truly on-chain unpredictable random sources.
 
-## Usage
+Ethereum PoS introduces randomness using block.mixHash (prevRandao). Look at [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399) for more information.
+
+## Installation
 
 You can install any of the extensions in this repository by running the following command:
```

Comment:
I think we can delete this sentence and just leave the command

Reply:
Yeah, seems irrelevant, I keep it just because we leave this sentence on the README from other extensions.